### PR TITLE
[7.0][Web API] Add documentation for active notifications API

### PR DESCRIPTION
**Pull request by m-bialek**
### Change Description ###

Add documentation for active notifications API:
- add new type of notifications to the guide
- edit mobile and wearable HTMLs

### API Changes ###

 - ACR: TWDAPI-284

### CODE REVIEW

**Review by pkosko — APPROVED** [review]
Looks good to me.

**Review by omar-srbd — COMMENTED** [review]
Hello @m-bialek,
I hope you are doing well.
I have been assigned to review this PR. It seems that you have made considerable changes on the files, shall I proceed to check all the contents of the files or only the parts that you have committed?
Please let me know.
Thanks.

**Comment by m-bialek** [comment]
Hi @omar-srbd! Both HTMLs have been generated automatically. Therefore, you could check the entire files for their consistency and errors I missed. I believe in the guide file (notification.md) you can check only the new parts.

**Review by omar-srbd — COMMENTED** [review]
Hello @m-bialek,
According to your instruction, I have checked both the HTML files and only the changed part of the .md file. 
I have made adequate corrections/suggestions wherever I could on the HTML files. However, there were several instances where I could not comment,
I am listing them below:

File: **docs/application/web/api/7.0/device_api/mobile/tizen/notification.html**
1.	Line 22 – “for accessing only internal……must be provided,”
Suggestion – “For accessing only internal……must be provided.”
Reason – The first word of a sentence should be capitalized and sentence should end with a period.
2.	Line 24 - “for accessing only external… must be provided,”
Suggestion – “For accessing only external… must be provided.”
Reason – The first word of a sentence should be capitalized and sentence should end with a period.
3.	Line 26 - “for accessing internal and external…”
Suggestion – “For accessing internal and external…”
Reason - The first word of a sentence should be capitalized.
4.	Line 28 - “…of asking user directly with proper…”
Suggestion – “…of asking the user directly with the proper…”
Reason – Addressed article usage.
5.	Line 37 - “Table of Contents”
Suggestion – “Table of contents”
Reason – Second headings should be written in a sentence case capitalization.
6.	Line 99 - “Summary of Interfaces and Methods”
Suggestion – “Summary of interfaces and methods”
Reason –Please use sentence-style capitalization for second headings.
 7.	Line 185 - “Type Definitions”
Suggestion – “Type definitions”
Reason - Please use sentence-style capitalization for second headings.
8.	Line 258 - “ "BYTE" …”
Suggestion – “BYTE…”
Reason – Should the word "BYTE" be enclosed in quotation marks here? As on previous lines, the keywords described in quotation marks in the code has been written without quotation marks. Please look into this.
9.	Line 260 - “ "PERCENTAGE" …”
Suggestion – “PERCENTAGE…”
Reason - Should the word "BYTE" be enclosed in quotation marks here? As on previous lines, the keywords described in quotation marks in the code has been written without quotation marks. Please look into this.
10.	Line 268 - “The following values are supported in this release :”
Suggestion – “The following values are supported in this release:”
Reason – Removal of the space before comma.
11.	Line 277 - “LED_CUSTOM_DUTY_ON, blink LED”
Suggestion – “LED_CUSTOM_DUTY_ON - blink LED.”
Reason – Consistency with other lines & period at the end of the sentence.
12.	Line 279 - “LED_CUSTOM_DEFAULT, default flag”
Suggestion – “LED_CUSTOM_DEFAULT - default flag.”
Reason – Consistency with other lines & period at the end of the sentence.
13.	_Line 384, 514, 573, 630, 791, 860, 916, 996 - “public”_
Suggestion – “Public”
Reason – The first word after a colon should always be capitalized.
14.	_Line 402, 406, 409, 412 – First word capitalize._
15.	_Line 406, 536 - “…application does not have privilege…”_
Suggestion – “…application does not have the privilege…”
Reason/Reference – Article missing addressed.
16.	_Line 532, 536, 539, 542 - First word capitalize._
17.	_Line 591, 594, 597, 600 - First word capitalize._
18.	_Line 639, 642 - First word capitalize._
19.	_Line 690, 693, 696 - First word capitalize._
20.	_Line 693, 742 - “…application does not have privilege…”_
Suggestion – “…application does not have the privilege…”
Reason – Article missing.
21.	Line 739, 742 - First word capitalize.
22.	Line 785 - “…milliseconds, RGBA color and… is assumed to have value 0xff). In this case only…”
Suggestion – “…milliseconds, RGBA color, and… is assumed to have a value of 0xff). In this case, only…”
Reason - Addressed article usage.
23.	_Line 821, 824, 827, 830 - First word capitalize._
24.	_Line 941, 944, 947, 950 - First word capitalize._
25.	_Line 1021, 1024, 1027- First word capitalize._
26.	Line 1021 - “…the application does not have privilege to…”
Suggestion – “…the application does not have the privilege to…”
Reason - Addressed article usage.
27.	Line 1024 - “…find a template with given name.”
Suggestion – “…find a template with a given name.”
Reason - Addressed article usage.
28.	Line 1191 - “…does not support related feature (e.g. LED notification settings).”
Suggestion – “…does not support the related features (e.g. LED notification settings).”
Reason/Reference - Addressed article usage.
29.	Line 1229 – Period missing at the end of the sentence.
30.	Line 1237 - “…value for its job…”
Suggestion – “…value for their job…”
31.	Line 1242 - “Range of…”
Suggestion – “The range of…”
Reason/Reference - Any particular reason why the "Range" has been capitalized here?
32.	_Line 1319, 1565- “The content to show when the option to display content is off in the Settings.”_
Suggestion – “The content to show when the option to display content is off in the settings.”
33.	Line 1604 - “…the device has notification LED.”
Suggestion – “…the device has a notification LED.”
Reason/Reference - Addressed article usage.
34.	Line 1871 - “Defines additional thumbnails-related settings of a notification.”
Suggestion – “Defines additional thumbnail-related settings of a notification.”
35.	Line 1906 - “Defines additional actions-related settings of a notification.”
Suggestion – “Defines additional action-related settings of a notification.”
36.	Line 2207 - “…device which has a led”
Suggestion – “…device that has an LED”
Reason - Addressed article usage.


File: **docs/application/web/api/7.0/device_api/wearable/tizen/notification.html**
1.	Line 23 – “for accessing only internal……must be provided,”
Suggestion – “For accessing only internal……must be provided.”
Reason – The first word of a sentence should be capitalized and sentence should end with a period.
2.	Line 25 - “for accessing only external… must be provided,”
Suggestion – “For accessing only external… must be provided.”
Reason – The first word of a sentence should be capitalized and sentence should end with a period.
3.	Line 27 - “for accessing internal and external…”
Suggestion – “For accessing internal and external…”
Reason - The first word of a sentence should be capitalized.
4.	Line 29 - “…of asking user directly with proper…”
Suggestion – “…of asking the user directly with the proper…”
Reason – article usage missing.
5.	Line 38 - “Table of Contents”
Suggestion – “Table of contents”
Reason – Second headings should be written in a sentence case capitalization.
6.	Line 97 - “Summary of Interfaces and Methods”
Suggestion – “Summary of Interfaces and methods”
Reason –Please use sentence style capitalization for second headings.
7.	Line 181 - “Type Definitions”
Suggestion – “Type definitions”
Reason - Please use sentence style capitalization for second headings.
8.	Line 254 - “ "BYTE" …”
Suggestion – “BYTE…”
Reason – Should the word "BYTE" be enclosed in quotation marks here? As on previous lines, the keywords described in quotation marks in the code has been written without quotation marks. Please look into this.
9.	Line 256 - “ "PERCENTAGE" …”
Suggestion – “PERCENTAGE…”
Reason - Should the word "BYTE" be enclosed in quotation marks here? As on previous lines, the keywords described in quotation marks in the code has been written without quotation marks. Please look into this.
10.	_Line 359, 489, 548, 605, 769, 849 - “public”_
Suggestion – “Public”
Reason – The first word after a colon should always be capitalized.
11.	_Line 377, 381, 384, 387 – First word capitalize._
12.	_Line 381, 511 - “…application does not have privilege…”_
Suggestion – “…application does not have the privilege…”
Reason – Article missing addressed.
13.	_Line 507, 511, 514, 517 - First word capitalize._
14.	_Line 566, 569, 572, 575 - First word capitalize._
15.	_Line 614, 617 - First word capitalize._
16.	_Line 665, 668, 671 - First word capitalize._
17.	_Line 668, 717 - “…application does not have privilege…”_
Suggestion – “…application does not have the privilege…”
Reason – Article missing.
18.	_Line 714, 717 - First word capitalize._
19.	_Line 794, 797, 800, 803 - First word capitalize._
20.	_Line 874, 877, 880 - First word capitalize._
21.	Line 874 - “…the application does not have privilege to…”
Suggestion – “…the application does not have the privilege to…”
Reason - Addressed article usage.
22.	Line 877 - “…find a template with given name.”
Suggestion – “…find a template with a given name.”
Reason - Addressed article usage.
23.	Line 1044 - “…does not support related feature (e.g. LED notification settings).”
Suggestion – “…does not support the related features (e.g. LED notification settings).”
Reason - Addressed article usage.
24.	Line 1082 – Period missing at the end of the sentence.
25.	Line 1090 - “…value for its job…”
Suggestion – “…value for their job…”
Reason - Addressed article usage.
26.	Line 1095 - “Range of…”
Suggestion – “The range of…”
Reason - Addressed article usage.
27.	_Line 1172, 1418 - “The content to show when the option to display content is off in the Settings.”_
Suggestion – “The content to show when the option to display content is off in the settings.”
Reason - 
28.	Line 1457 - “…the device has notification LED.”
Suggestion – “…the device has a notification LED.”
Reason - Addressed article usage.
29.	Line 1724 - “Defines additional thumbnails-related settings of a notification.”
Suggestion – “Defines additional thumbnail-related settings of a notification.”
Reason - Addressed article usage.
30.	Line 1759 - “Defines additional actions-related settings of a notification.”
Suggestion – “Defines additional action-related settings of a notification.”
31.	Line 2060 - “…device which has a led”
Suggestion – “…device that has an LED”
Reason - Addressed article usage.

**Comment by m-bialek** [comment]
Thank you @omar-srbd for your review. I the new commit I fixed only the parts added by this PR. All other suggestions are going to be applied in another PR.

File: docs/application/web/api/7.0/device_api/mobile/tizen/notification.html [hunk 1]
```diff
@@ -993,6 +1043,60 @@ <h4>Methods</h4>
...
+<p><span class="version">Since: </span>
+ 7.0
+            </p>
+<div class="description">
+            <p>
+The user can set the notification block state in settings. The block state indicates whether or not notifications can be posted.
+            </p>
+           </div>
+<p><span class="privilegelevel">Privilege level: </span>
+ public
```
[the remark this answers is not in the document]
Comment (m-bialek) [reply docs/application/web/api/7.0/device_api/mobile/tizen/notification.html:1063]: Unfortunately I cannot change this one as this form is used in all files. Good point though.

**Comment by mijin85cho** [comment]
Hello, @asm-foysal I hope you're doing well..

It seems this PR is not approved yet from SRBD reviewers. Did you had chat with @omar-srbd about merging this PR?

**Review by asm-foysal — COMMENTED** [review]
@mijin85cho hello! I'm doing well, thank you. 
Yes, I had a chat with Omar before merging this PR.

**Comment by mijin85cho** [comment]
> Reference in ne

I see. Thanks for confirming.
To prevent the confusion between members as like [my previous comment](https://github.com/Samsung/tizen-docs/pull/1601#issuecomment-1097730000), please leave the traces on github.
When you merge the PRs, please check if they are 'approved'.

**Pull request merged**